### ISSUE TITLE
[BUG] Windows detects the .exe file as a virus

### ISSUE THREAD

**Issue by stefandulus** [casualsnek/onthespot#151]
When i try to open the exe, windows smartscreen pops up. help

**Comment by justin025** [comment]
which exe

**Comment by stefandulus** [comment]
the onthespot_win.exe

**Comment by justin025** [comment]
Probably a false positive

**Comment by Pippa015** [comment]
Yes, this is normal. To get rid of that popup the creator of OnTheSpot would need to pay a license to Microsoft which is pretty expensive if I remember correctly, so it's easier to just not pay it. When the smartscreen pops up, press "more info" (the small text under the main paragraph) and then "run anyways".
